setup.py

```diff
@@ -23,6 +23,9 @@
                 'Programming Language :: Python :: 2.5',
```

Comment:
And do you want to keep the 2.5 classifier. AFAIK you don't advertise it working with 2.5.